testing_config.py

```diff
@@ -67,6 +67,7 @@ def setUpOurTestbed():
   ourTestbed.init_urlfetch_stub()
   ourTestbed.init_user_stub()
   ourTestbed.init_xmpp_stub()
+  ourTestbed.init_memcache_stub()
```

Comment:
The not-so-old NDB client library uses memcache by default for caching. Hence, we need it while running tests. However, the Cloud NDB library would not use memcache.

Reply:
OK that makes sense.  I look forward to being able to remove this line again.